_sass/jekyll-theme-architect.scss

```diff
@@ -440,62 +436,58 @@ span.requisites {
 .tooltip {
     position: relative;
     display: inline-block;
-}
-
-/* Tooltip text */
-.tooltip .tooltiptext {
-    visibility: hidden;
-    background-color: #666;
-    color: #ddd;
-    text-align: left;
-    padding: 8pt;
-    border-radius: 4pt;
-    font-size: 10pt;
-    font-weight: 200;
-
-    top: 115%;
-    left: 18pt; 
-    width: 200pt;
-    position: absolute;
-    z-index: 1;
-}
-
-.tooltip .tooltiptext::after {
-    content: " ";
-    position: absolute;
-    bottom: 100%;
-    left: 10%;
-    margin-left: -5px;
-    border-width: 5px;
-    border-style: solid;
-    border-color: transparent transparent #666 transparent;
-}
 
-.tooltip:hover .tooltiptext {
-    visibility: visible;
+    &:hover .tooltiptext {
+      visibility: visible;
+    }
+
+    .tooltiptext {
+      visibility: hidden;
+      background-color: #666;
+      color: #ddd;
+      text-align: left;
+      padding: 8pt;
+      border-radius: 4pt;
+      font-size: 10pt;
+      font-weight: 200;
+
+      top: 115%;
+      left: 18pt;
+      width: 200pt;
+      position: absolute;
+      z-index: 1;
+
+      &:after {
+        content: " ";
+        position: absolute;
+        bottom: 100%;
+        left: 10%;
+        margin-left: -5px;
+        border-width: 5px;
+        border-style: solid;
+        border-color: transparent transparent #666 transparent;
+      }
+    }
 }
 
 /* MISC */
-.clearfix:after {
-  display: block;
-  height: 0;
-  clear: both;
-  visibility: hidden;
-  content: '.';
-}
 
 .clearfix {
-  display: inline-block;
```

Comment:
I changed this because few lines after this rule was overridden with `display: block`